hack/tests/subcommand-scorecard.sh

```diff
@@ -28,12 +29,27 @@ else
 	exit 1
 fi
 
+# test to see if bundle flag works correctly
```

Comment:
wdyt about we use header_text here? https://github.com/operator-framework/operator-sdk/blob/master/hack/common.sh#L17

```suggestion
header_text 'testing operator-sdk scorecard --bundle to see if bundle flag works correctly'
```

Reply:
good idea, I added header_text instead of the comments for each test.